scripts/download_json/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+/**
+ * Random scripts
+ * @module Scripts
+ */
+
+/**
+ * Generates the string to use as download link of a object in JSON format
+ *
+ *
+ * @param {object} Object - Object that will be downloaded as JSON
```

Comment:
between the braces you write the Type of variable and after that, the name of the variable. In this case, we should change `{object}` to `{Object}` capitalized